dm_cli/import_package.py

```diff
@@ -133,6 +133,10 @@ def upload_global_file(address: str) -> str:
         filepath = Path(address)
         if filepath.suffix != ".json":
             # Binary files
+            if global_files is None:
+                raise ApplicationException(
+                    f"Attempted to upload global files {filepath}, but no global folder was defined in the datasource definition..."
```

Comment:
How can you know how the datasource definition looks like based on global_files being "none"?

Reply:
global_files are created based on the datasource definition and passed down. if no global folder was defined in the data source, the zip file will be empty and you get an error if you try to find anything in it